homeassistant/components/camera/__init__.py

```diff
@@ -783,6 +785,9 @@ def async_update_token(self) -> None:
     async def async_internal_added_to_hass(self) -> None:
         """Run when entity about to be added to hass."""
         await super().async_internal_added_to_hass()
+        self.__supports_stream = (
+            self.supported_features_compat & CameraEntityFeature.STREAM
+        )
         await self.async_refresh_providers(write_state=False)
```

Comment:
Can we remove this line now?

Reply:
Yes, I also think it makes sense to keep it here, then the lion's share of cameras will work without an extra refresh.